benchopt/utils/sys_info.py

```diff
@@ -26,13 +26,20 @@ def _get_processor_name():
 
 
 def _get_cuda_version():
```

Comment:
I would now make this function public as we use it in benchmark code

Reply:
Good call